### PR TITLE
Add Box achievements

### PR DESCRIPTION
**Pull request by theofidry**

### CODE REVIEW

File: achievements/box.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Box Achievements
+
+[Humbug Box][box] is an application bunlder for PHP apps. It allows one to ship a PHP app into a PHAR and distribute it with
```
Comment (greg0ire) [inline achievements/box.md]:
```suggestion
[Humbug Box][box] is an application bundler for PHP apps. It allows one to ship a PHP app into a PHAR and distribute it with 
```

File: achievements/box.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+[Humbug Box][box] is an application bunlder for PHP apps. It allows one to ship a PHP app into a PHAR and distribute it with 
+ease.
+
+Over the last few months, Box has been growing in popularity especially within the [Laravel][laravel] community thanks to the
+[Laravel Zero][laravel-zero] framework, a spin-off of Laravel more focused on CLI applications.
+
+In contrast, while Box is being used a lot with projects making use of [Symfony][symfony] components, it cannot be used, at
+least easily with a standard Symfony application.
+
+Thanks to this hackaton, [I][me] could besides working on other Symfony related issues, and with the help of
```
Comment (greg0ire) [inline achievements/box.md]:
```suggestion
Thanks to this hackaton, [I][me] could, besides working on other Symfony related issues, and with the help of 
```

File: achievements/box.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+In contrast, while Box is being used a lot with projects making use of [Symfony][symfony] components, it cannot be used, at
+least easily with a standard Symfony application.
+
+Thanks to this hackaton, [I][me] could besides working on other Symfony related issues, and with the help of 
+[@nicolas-grekas][nicolas], [@stof][stof] and [@tucksaun][tucksaun], work on adding support for Symfony for Box.
+
+The result was:
+
+- The realease of [Box 3.7.0][box-3.7.0]
```
Comment (greg0ire) [inline achievements/box.md]:
```suggestion
- The release of [Box 3.7.0][box-3.7.0]
```

File: achievements/box.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+ease.
+
+Over the last few months, Box has been growing in popularity especially within the [Laravel][laravel] community thanks to the
+[Laravel Zero][laravel-zero] framework, a spin-off of Laravel more focused on CLI applications.
+
+In contrast, while Box is being used a lot with projects making use of [Symfony][symfony] components, it cannot be used, at
+least easily with a standard Symfony application.
+
+Thanks to this hackaton, [I][me] could besides working on other Symfony related issues, and with the help of 
+[@nicolas-grekas][nicolas], [@stof][stof] and [@tucksaun][tucksaun], work on adding support for Symfony for Box.
```
Comment (greg0ire) [inline achievements/box.md]:
```suggestion
[@nicolas-grekas][nicolas], [@stof][stof] and [@tucksaun][tucksaun], work on adding support for Symfony to Box.
```

File: achievements/box.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+The result was:
+
+- The realease of [Box 3.7.0][box-3.7.0]
+- 11 issues closed
+- 14 pull requests
+
+This work also resulted directly into:
+
+- 2 pull requests made on the Symfony repositories (recipes & core) regarding the CLI guards within the Symfony codebase
+- Add additional screening on Symfony regarding the [HTTPoxy][httpoxy] vulnerability (not security issue found during the process)
```
Comment (greg0ire) [inline achievements/box.md]:
```suggestion
- Adding additional screening on Symfony regarding the [HTTPoxy][httpoxy] vulnerability (no security issue found during the process)
```

File: achievements/box.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+- 1 issue with a reproducer regarding the [Symfony Console component][symfony-console]
+- 1 issue reproduced and solved for the [Symfony CLI tool][symfony-cli]
+
+As well more discussions have been made on how to integrate a [REPL][repl] to Symfony with minimal dependencies (which has been
```
Comment (greg0ire) [inline achievements/box.md]:
```suggestion
Also, more discussions have been made on how to integrate a [REPL][repl] to Symfony with minimal dependencies (which has been
```

**Review by greg0ire — CHANGES_REQUESTED** [review]

**Review by Simperfit — APPROVED** [review]
after resolving @greg0ire comments :+1:

File: achievements/box.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+[Humbug Box][box] is an application bundler for PHP apps. It allows one to ship a PHP app into a PHAR and distribute it with 
+ease.
+
+Over the last few months, Box has been growing in popularity especially within the [Laravel][laravel] community thanks to the
+[Laravel Zero][laravel-zero] framework, a spin-off of Laravel more focused on CLI applications.
+
+In contrast, while Box is being used a lot with projects making use of [Symfony][symfony] components, it cannot be used, at
+least easily with a standard Symfony application.
+
+Thanks to this hackaton, [I][me] could, besides working on other Symfony related issues, and with the help of
```
Comment (greg0ire) [inline achievements/box.md]:
```suggestion
Thanks to this hackathon, [I][me] could, besides working on other Symfony related issues, and with the help of 
```

**Review by greg0ire — APPROVED** [review]

File: achievements/box.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+[laravel]: http://laravel.com
+[laravel-zero]: https://laravel-zero.com/
+[symfony]: https://symfony.com/
+[me]: https://github.com/theofidry
+[nicolas]: https://github.com/nicolas-grekas
+[tucksaun]: https://github.com/tucksaun
+[box-3.7.0]: https://github.com/humbug/box/releases/tag/3.7.0
+[httpoxy]: https://httpoxy.org
+[symfony-console]: https://symfony.com/doc/current/components/console.html
+[repl]: https://es.wikipedia.org/wiki/REPL
```
Comment (Toflar) [inline achievements/box.md]:
```suggestion
[repl]: https://en.wikipedia.org/wiki/Read–eval–print_loop
```

**Review by Toflar — APPROVED** [review]

File: achievements/box.md [hunk 9]
```diff
@@ -0,0 +1,45 @@
+# Box Achievements
+
+[Humbug Box][box] is an application bundler for PHP apps. It allows one to ship a PHP app into a [PHAR][phar] and distribute
+it with ease.
+
+Over the last few months, Box has been growing in popularity especially within the [Laravel][laravel] community thanks to the
+[Laravel Zero][laravel-zero] framework, a spin-off of Laravel more focused on CLI applications.
+
+In contrast, while Box is being used with lot with projects making use of [Symfony][symfony] components, it cannot be used – at
```
Comment (stof) [inline achievements/box.md]: typo. Should be `a lot`

File: achievements/box.md [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+- Adding additional screening on Symfony regarding the [HTTPoxy][httpoxy] vulnerability (no security issue found during the process)
+- 1 issue with a reproducer regarding the [Symfony Console component][symfony-console]
+- 1 issue reproduced and solved for the [Symfony CLI tool][symfony-cli]
+
+Also, more discussions have been made on how to integrate a [REPL][repl] to Symfony with minimal dependencies (which has been
+a recurring problem so far) thanks to [PsySH][psysh] and [Box][box].
+
+
+[box]: https://github.com/humbug/box
+[laravel]: http://laravel.com
```
Comment (stof) [inline achievements/box.md]: should be HTTPS

File: achievements/box.md [hunk 11]
```diff
@@ -6,7 +6,7 @@ it with ease.
 Over the last few months, Box has been growing in popularity especially within the [Laravel][laravel] community thanks to the
 [Laravel Zero][laravel-zero] framework, a spin-off of Laravel more focused on CLI applications.
 
-In contrast, while Box is being used with lot with projects making use of [Symfony][symfony] components, it cannot be used – at
+In contrast, while Box is being used with a lot with projects making use of [Symfony][symfony] components, it cannot be used – at
```
Comment (stof) [inline achievements/box.md]: The extra `with` should also be removed

File: achievements/box.md [hunk 12]
```diff
@@ -0,0 +1,45 @@
+# Box Achievements
+
+[Humbug Box][box] is an application bundler for PHP apps. It allows one to ship a PHP app into a [PHAR][phar] and distribute
+it with ease.
+
+Over the last few months, Box has been growing in popularity especially within the [Laravel][laravel] community thanks to the
+[Laravel Zero][laravel-zero] framework, a spin-off of Laravel more focused on CLI applications.
+
+In contrast, while Box is being used a lot with projects making use of [Symfony][symfony] components, it cannot be used – at
```
Comment (greg0ire) [inline achievements/box.md]: You removed the wrong with ;)
```suggestion
In contrast, while Box is being used a lot with projects making use of [Symfony][symfony] components, it cannot be used – at
```

File: achievements/box.md [hunk 13]
```diff
@@ -0,0 +1,45 @@
+# Box Achievements
```
Comment (jfhovinne) [inline achievements/box.md]: There is on ongoing discussion in #20 about capitalisation in titles. If we follow the EC style guide, only the first word takes a capital.

File: achievements/box.md [hunk 14]
```diff
@@ -0,0 +1,45 @@
...
+it with ease.
+
+Over the last few months, Box has been growing in popularity especially within the [Laravel][laravel] community thanks to the
+[Laravel Zero][laravel-zero] framework, a spin-off of Laravel more focused on CLI applications.
+
+In contrast, while Box is being used with a lot of projects making use of [Symfony][symfony] components, it cannot be used – at
+least easily, with a standard Symfony application.
+
+Thanks to this hackathon, [I][me] could, besides working on other Symfony related issues, and with the help of 
+[@nicolas-grekas][nicolas], [@stof][stof] and [@tucksaun][tucksaun], work on adding support for Symfony to Box.
```
Comment (jfhovinne) [inline achievements/box.md:13]: stof link does not display correctly (see rendered page), missing in the list below

**Review by jfhovinne — CHANGES_REQUESTED** [review]

**Comment by theofidry** [comment]
@jfhovinne done

**Review by jfhovinne — APPROVED** [review]
Great, thanks! I see the link from homepage is missing, but will add it.

**Pull request merged**